crud/borders.lua

```diff
@@ -186,7 +186,7 @@ end
 -- @param string space_name
 --  A space name
 --
--- @param ?string index_name
+-- @param ?string index_id
 --  An index name (by default, primary index is used)
```

Comment:
The parameter is called ``index_id``, but below it is described as ``An index name (by default, primary index is used)``. This is confusing

Reply:
Agree. Fixed and pushed.